sympy/printing/pretty/pretty.py

```diff
@@ -218,14 +218,20 @@ def _print_binomial(self, e):
 
         return pform
 
-    def _print_Relational(self, e):
-        op = prettyForm(' ' + xsym(e.rel_op) + ' ')
+    def _helper_print_relations(self, lhs, rel_op, rhs):
+        op = prettyForm(' ' + xsym(rel_op) + ' ')
```

Comment:
This currently fails with a KeyError.

Reply:
Can you provide an example? It works fine in Jupyter and on the command line for me. What version of python3?